tools/myst/enc/Makefile

```diff
@@ -47,7 +47,12 @@ LIBS += $(LIBDIR)/libmystgcovclient.a
 endif
 
 LDFLAGS += $(OEENCLAVE_LDFLAGS)
-LDFLAGS += -lcrypto
+# The following workaround can be removed once internal crypto
+# implementation is ported to use OpenSSL
+LDFLAGS += -lmbedx509
```

Comment:
Should we do the same for `target/sgx/enclave`?

Reply:
No. This is only required for linking (when linking against `libmysttargetsgxenclave.a`).